src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2721,6 +2729,10 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
          * Data files are external so data files should not be loaded into problem data files.
          */
         boolean loadExternalFile = problem.isUsingExternalDataFiles();
+        
+        boolean loadSamples = contest.getContestInformation().isLoadSampleJudgesData();
+        
+        String sampleDataDirectory = dataFileBaseDirectory.replaceAll("secret$", "sample");
```

Comment:
These lines of code are confusing; they get the boolean value `loadSamples` and the name of the `sampleDataDirectory`, but then those values are not used in the following lines (which use `dataFileBaseDirectory` instead, and make no use of `loadSamples`).   It would be much clearer if these lines were moved down just in front of line 2756, which is the place where these values are actually used.

Reply:
Again, not something that should hold up approval... I just don't care for adding code that's confusing (even if it executes correctly) when it could be avoided...